operator/cmd/mesh.go

```diff
@@ -1,28 +0,0 @@
-// Copyright 2019 Istio Authors
```

Comment:
Is this still in use? There's a whole `mesh` subdir that could be removed as well if this is the case.
`git grep mesh.go` returns some references to this file. Better to remove this and all those references in a separate PR?

Reply:
Yeah I added it back for now, it's being used for e2e tests somewhere. Will remove in a future PR